src/FileMigrationHelper.php

```diff
@@ -168,28 +170,38 @@ protected function validateFileClassname($file, $extension)
      */
     protected function getFileQuery()
     {
+        $table = DataObject::singleton(File::class)->baseTable();
         // Select all records which have a Filename value, but not FileFilename.
         /** @skipUpgrade */
         return File::get()
             ->exclude('ClassName', [Folder::class, 'Folder'])
             ->filter('FileFilename', array('', null))
-            ->where('"File"."Filename" IS NOT NULL AND "File"."Filename" != \'\''); // Non-orm field
+            ->where(sprintf(
+                '"%s"."Filename" IS NOT NULL AND "%s"."Filename" != \'\'',
+                $table, $table
+            )) // Non-orm field
+            ->alterDataQuery(function (DataQuery $query) use ($table) {
+                return $query->addSelectFromTable($table, ['Filename']);
+            });
     }
 
     /**
      * Get map of File IDs to legacy filenames
      *
+     * @deprecated 4.4..5.0
```

Comment:
Minor: PSR-19 specifies a semver constraint "from", and they've removed the "to" part (https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md#54-deprecated)

```suggestion
     * @deprecated 4.4.0
```